src/http/controllers/health.rs

```diff
@@ -0,0 +1,36 @@
+use saphir::*;
+use saphir::Method;
+
+struct ControllerData {
+}
+
+pub struct HealthController {
+    dispatch: ControllerDispatch<ControllerData>,
+}
+
+impl HealthController {
+    pub fn new() -> Self {
+        let dispatch = ControllerDispatch::new(ControllerData{});
+        dispatch.add(Method::GET,
+                     "/",
+                     health);
+
+        HealthController {
+            dispatch
+        }
+    }
+}
+
+impl Controller for HealthController {
+    fn handle(&self, req: &mut SyncRequest, res: &mut SyncResponse) {
+        self.dispatch.dispatch(req, res);
+    }
+
+    fn base_path(&self) -> &str {
+        "/health"
+    }
+}
+
+fn health(_: &ControllerData, _req: &SyncRequest, res: &mut SyncResponse) {
+    res.status(StatusCode::OK).body("I'm here and I'm alive, that's enough");
```

Comment:
Maybe a check on what's going on inside the server could be insightful here (e.g. Check database connection)

Reply:
Yeah, I looked what I could do, but we don't have a database and the server is only a listener on 8080 to create a proxy between 2 wayk.  If the program is running, it means that we are still listening on 8080 and the forward future are polled... So I didn't see anything that could be useful to check. I added that route just to have it in future if we would like to check a database connection for example, as you said.

Reply:
Yeah, but the fact that the program is still running doesn't ensure that your still listening on 8080

Reply:
If something happen, I guess an error will be raised and futures will finish.  Or is there a way to know that we are still listening and processing request ?